### ISSUE TITLE
Reorganizing Site Content

### ISSUE THREAD

**Issue by kyscg** [kyscg/kyscg.github.io#3]
Partly because it is necessary, mostly because of my perpetual dissatisfaction with what I create; I find myself — for the hundred and twenty-eighth time — wanting to revamp the website. _Primo_, the [notes](https://kyscg.github.io/notes) page is barren, and untouched for over two years. Traffic to the page is limited to bots scraping the pdf files that I've linked to there. All this is very inelegant, and inefficient. _Secundo_, the notes page should be what it says, short-form or long-form **notes**. The [about](https://kyscg.github.io/about) and [reading](https://kyscg.github.io/reading) page also could do with some serious thinking. As much as possible, I would prefer to preserve the five page structure of the website.

I have some ideas for how to go about all this and I'll list down steps as and when I think of them.

- [x] Shift content in notes to articles appropriately
- [ ] Convert pdf file attachments to html pages `??`
- [x] Spell out research interests and directions better in [about](https://kyscg.github.io/about)
- [x] All the writing in [about](https://kyscg.github.io/about) is unnecessary; delete, delete, delete.
- [ ] This will take a lot of effort, but add reviews to all the books in [reading](https://kyscg.github.io/reading) `??`
- [x] Timestamped, linkable notes (could be short-form, long-form, images, embeds, links et cetera)
- [x] Integrate [todo](https://kyscg.github.io/todo) into notes.
- [ ] Create RSS feed for the site

I will either add to the list if I have additional ideas, mark as completed with a corresponding commit or strike-through in case I don't want to do it or want to do it in a separate branch. the changes appended with `??` will mostly not be done, unless I can find a way to not do a significant part of them manually.

As usual, all ideas are welcome. I'm not very sure about how to get the notes to be linkable, and I'll need to experiment a lot, so this will take a few weeks, maybe months, before I iron out everything. May Pallas Athena lead the way.

**Comment by kyscg** [comment]
Did the following in https://github.com/kyscg/kyscg.github.io/pull/4/commits/5c332cc762592267391d34ab86c3d3b74254d634

- Moved over all links from Notes
- Added dates to navbar in justiceharvard.html and removed the updated entry because an ugly looking scroll bar appears every time one loads the page
- Added UFO and WIP tags to posts, and a tiny explanation at the bottom of the page.
- Added newton method gist to homepage

**Comment by kyscg** [comment]
Did the following in https://github.com/kyscg/kyscg.github.io/pull/4/commits/b09ebf79ae356a2f7442a9521e589d27744efafa

- Complete overhaul of About
    - Research interests and personal interests listed more neatly
    - Added site colophon
    - Removed unnecessary links and cleaned up contact section
- Made design changes to blockquotes to make them stand out better.
- Updated reading.html with latest books and reviews.

**Comment by kyscg** [comment]
Did the following in https://github.com/kyscg/kyscg.github.io/pull/4/commits/4a992c96234723209abe1e687cf24a3348fb1c0b

- Notes is brand new and sparkling
    - Every note is it's own linkable card (although it isn't really obvious when you look at the page)
    - Notes are mainly text, math, quotes, link dumps, or embeds
- Design changes
    - Image margins are now set to `auto`
    - CSS styling for cards in notes.html
    - CSS styling for image grid support
- Changes to index.html
    - Removed [WIP] tags from posts that are completed
    - Added links to audio gist, lecture notes on virology and death
- Changes to about.html
    -  Added design principles and a blogroll
    - Added cal.com link in contact section
    - Rewrote some of the text to make it less cringey
- Snipped everything away from todo.md and integrated important stuff into notes.html
- Images in `assets/images/notes` are named by the date of the note
- 404.html now has a confused cat
- Updated reading.html with latest books and reviews

**Comment by kyscg** [comment]
I finally got it done, on the 108th day of asking. This is more proof to the fact if you don't have scheduled maintenance, you're going to be hit with unscheduled malfunction. I gave up on the tasks unchecked because they were taking too long. I must've spent at least four hours trying to get RSS to work and only had a headache to show for my efforts. I've pushed all of them to https://github.com/kyscg/kyscg.github.io/issues/5 and I plan to never think of them again.

Happy Ides of March!
![idesofmarch](https://github.com/kyscg/kyscg.github.io/assets/35304641/0a379e13-3eca-4559-81aa-6a2cc6d917ac)
